include/config.hpp

```diff
@@ -0,0 +1,37 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   config.hpp                                         :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: lizhang <lizhang@student.codam.nl>           +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2023/06/26 16:49:27 by lizhang       #+#    #+#                 */
+/*   Updated: 2023/06/26 16:49:27 by lizhang       ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef CONFIG_HPP
+#define CONFIG_HPP
+
+#include <string>
+
+class Config_mac
```

Comment:
I would just name it config, no need for the mac, since we already check if the os is mac or not.